web/vibe/web/internal/rest/common.d

```diff
@@ -300,7 +301,7 @@ import std.traits : hasUDA;
 					pi.isOut = true;
 				} else static if (SC & ParameterStorageClass.ref_) {
 					pi.isIn = true;
-					pi.isOut = true;
+					pi.isOut = isMutable!PT;
```

Comment:
Let's see if this will help. Works fine on LDC 1.27.0.

Reply:
Hold on, we use `inIn` for "input parameter", not the `in` storage class, right ? Damn that's confusing :D